### DISCUSSION TITLE
More Pets and Mounts in shop

### DISCUSSION THREAD

**Discussion by Randomdruid** [chromiecraft/chromiecraft#4518] [Suggestions]
Could you possibly add more mounts and pets to the shop? The rarer the better.

Thank you

**Comment by balleny** [comment]
Why dont you provide a list of mounts and pets you have in mind?

**Comment by Randomdruid** [comment]
Because theres alot, over 200 mounts in the game at least.

Some examples like:

Naxxramas Deathcharger Reins
Reins of the Black Warp Stalker
Reins of the Black Polar Bear
Reins of the Brown Polar Bear
Great Elite Elekk
Reins of the Swift Dawnsaber

**Comment by Nyeriah** [comment]
Warpstalker doesnt seem to be a mount

![image](https://user-images.githubusercontent.com/47818697/204931124-7db210fd-4a77-40ab-8896-b7d24e9b3c42.png)


The polar bears were added to the game as rewards for the PvP faction leader achievements, Stormpeaks daily quest satchel (same model, different names)

**Comment by Amrx2** [comment]
Hi Randomdruid, for us to consider adding a mount or pet in game it needs to be unattainable by normal means. If you know of some which are from TGC for example or in the files that you cannot earn in game let us know.

Cheers,
Construct
